examples/default.yaml

```diff
@@ -299,6 +299,10 @@ rosetta:
 # 🟢 Builtin default: use name from /etc/timezone or deduce from symlink target of /etc/localtime
 timezone: null
 
+# Allow using IPFS for downloading files with a provided CID.
+# 🟢 Builtin default: false
+ipfs: null
```

Comment:
Can we just use `location: cid://<CID>` ?

Reply:
Or `location: ipfs://<CID>` ?

Reply:
You _can_ use ipfs: instead of https:, but then it becomes a different URL.

https://github.com/lima-vm/lima/pull/2408#issuecomment-2180784234

Reply:
It is possible to do this with the current implementation, i.e. it supports both ways

You can change the location (url), or you can provide a cid for an existing location

Reply:
Since the fields are optional, they should not interfer so much with existing templates?

Probably want to have some more automation / sharing in place, before adding them.